walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -33,53 +38,102 @@
     // Hash map looks like {0=1=0, 1=-1=0, 2=1=0, 3=-1=0}.
     // +1 for 0 and 2, because A starts the day without a meeting/finishes the meeting
     // -1 for 1 and 3, because it's the end of the day/A goes to the meeting
-    HashMap<Integer, Long> timePointsOfAvailabilityChange = new HashMap<>();
-    Collection<String> attendees = request.getAttendees();
+    HashMap<Integer, Map.Entry<Long, Long>> timePointsOfAvailabilityChange = new HashMap<>();
```

Comment:
Can you please update the documentation now that the map is not `Int` -> `Long`? I'm curious to understand why you're using `Map.Entry<Long, Long>`. Maybe there is another structure that can be used here, but it will depend on what the values should represent

Reply:
I use Map.Entry because I need Pair, and as far as I know, Java don't have it in a standard library (which is extremely strange) like other normal languages))
If you could suggest me the alternative, it could be great, because I'm sure that the must be more adequate alternative. Maybe it makes sense to create a class for it, idk(

Reply:
The best way to do it in my opinion is to create a wrapper class. Something like:

```java
private class AvailabilityChange {
  long mandatoryAttendees;
  long optionalAttendees;
}
```

Then it gets much cleaner for both creation, i.e. `timePointsOfAvailabilityChange.getOrDefault(eventStartTime, new AvailabilityChange())` (you can even add a constructor if you need to initialize it with values different than 0L, 0L), and access, i.e. `timePointChanges.mandatoryAttendees` and `timePointChanges.optionalAttendees`